blocks/init/src/Blocks/components/share/assets/index.js

```diff
@@ -1,7 +1,8 @@
 import domReady from '@wordpress/dom-ready';
+import manifest from './../manifest.json';
 
 domReady(() => {
-	const shareLinkSelector = '.js-share-link';
+	const shareLinkSelector = `.${manifest.componentJsClass}`;
```

Comment:
why not destructure.

`import { componentJsClass } from '../manifest.json';`

Reply:
no reason. Fixed